scripts/data_collection/user_data_aggregator.py

```diff
@@ -423,13 +423,16 @@ def read_full_conversation_history(
                     timestamp = conversation["metadata"]["timestamp"]
                     query = conversation["redacted_query"].strip()
                     response = conversation["llm_response"].strip()
+                    rag_context = conversation["rag_context"].strip()
                     conversation_record = f"\nQ:{query}\nA:{response}\n"
 
                     if referenced_documents:
                         docs = format_referenced_documents(
                             conversation["referenced_documents"]
                         )
                         conversation_record += f"\nReferenced documents:\n{docs}\n"
+                    if rag_context:
+                        conversation_record += f"rag context:\n{rag_context}\n"
```

Comment:
With the changes in this PR, the output looks like this.
```
Q:what is pod
A:A pod is ... ile it is running.
----------------------------------------------------------------------------------------------------
RAG context part

Referenced doc:
Using pods: https://docs.openshift.com/container-platform/4.15/nodes/pods/nodes-pods-using.html
RAG context:
Using p.....
```

Reply:
Sure, AFAIK we don't really have any processing now. We can tailor it once there is a use case.
But if you want to have it in a specific format now, I can change of course.

Reply:
You can use `"RAG context part"` in split to get the rag. But I think for any processing outside of the spreadsheet, we should think about what we want to achieve and how (more structured data instead of parsing strings).